lib/rules/indentation/README.md

```diff
@@ -157,7 +157,7 @@ By default, the indent level of the CSS code block in non-CSS-like files is dete
 For example, with `[ 2, { baseIndentLevel: 1 } ]`, CSS should be indented 1 levels higher than `<style>` tag:
 
 ```html
-<!DOCTYPE html>
+<!doctype html>
```

Comment:
See https://prettier.io/blog/2023/07/05/3.0.0.html#print-html5-doctype-in-lowercase-7391httpsgithubcomprettierprettierpull7391-by-fiskerhttpsgithubcomfisker